src/js/models/v2/CoreLocation.js

```diff
@@ -64,7 +64,8 @@ const CoreLocation = {
     get fullAddress() {
         const line1 = (this.streetAddress && `${this.streetAddress}`) || '';
         const line2 = (this.regionalAddress && `${this.regionalAddress}\n`) || '';
-        return `${line1}${line2}${this.congressionalDistrict}` || '--';
+        const line3 = (this.congressionalDistrict && `Congressional District: ${this.congressionalDistrict}`) || '';
+        return `${line1}${line2}${line3}` || '--';
```

Comment:
This logic is fine, but it doesn't look like the `\n` are rendering on the actual page. I think we need to add `white-space: pre-wrap;` in the CSS to the relevant items.
https://developer.mozilla.org/en-US/docs/Web/CSS/white-space